Scripts/Mac/BuildGVFSForMac.sh

```diff
@@ -34,7 +34,7 @@ fi
 
 $SCRIPTDIR/DownloadGVFSGit.sh || exit 1
 GVFSPROPS=$SRCDIR/GVFS/GVFS.Build/GVFS.props
-GITVERSION="$(cat $GVFSPROPS | grep GitPackageVersion | grep -Eo '[0-9.]{1,}')"
+GITVERSION="$(cat $GVFSPROPS | grep GitPackageVersion | grep -Eo '[0-9.]+(-\w+)?')"
```

Comment:
I'm not familiar with bash regex syntax, what does `+(-\w+)?` match against?

Reply:
What does the `\w` match? (also where did you find the spec?)

Reply:
The \w matches any word (alphanumeric) character, so we're matching any number of words after the -. So -pr, -rc, -release1 would all be valid values, but something like -release-1 wouldn't be.

The 'spec' of sorts is here: https://docs.microsoft.com/en-us/nuget/reference/package-versioning